code/headers/uart_ports.hpp

```diff
@@ -1,50 +1,14 @@
 #pragma once
+
 #include <hwlib.hpp>
 
-namespace r2d2 {
-    
-    
+namespace r2d2::usart {
     enum class uart_ports_c {
-        uart0, uart1, uart2, uart3, 
+        uart0,
+        uart1,
+        uart2,
+        uart3,
         // only for determining how many uart ports there are
         UART_SIZE
-
```

Comment:
The only reason enum class uart_ports_c has its own file is because this commented code, it should probably be placed in usart_connection.hpp now

Reply:
sure I can move this to the usart_connection.hpp

Reply:
I have moved the usart_ports_c to the hardware_usart.hpp and renamed it in all the files